languages/go/exercises/concept/basics/.docs/instructions.md

```diff
@@ -29,11 +29,11 @@ PreparationTime(2)
 // => 4
 ```
 
-## 4. Calculate the elapsed working time in minutes
+### 4. Calculate the total working time in minutes
```

Comment:
This header is now smaller than the others...

Reply:
Also see my comment on `total` vs `elapsed` above. Interested in your thoughts!

Reply:
I've fixed the header as well.

Reply:
That's a really nice article. Thanks for sharing it!